packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -13,41 +15,80 @@ export let dismiss: () => void;
 export let action: { text: string; handler: () => unknown } | undefined =
   undefined;
 
+/** The severity of the notification you want to show users*/
+export let variant: ToastVariantType = 'success';
+
 /** Additional CSS classes to pass to the banner. */
 let extraClasses: cx.Argument = '';
 export { extraClasses as cx };
+
+let icon: IconName | null = null;
+let iconClasses = '';
+$: {
```

Comment:
Can this functionality of "select icon and text color based on variant" be moved back to `variants.ts`? For bonus points, we could add unit tests to `variants.ts`, too